packages/flutter_tools/test/integration.shard/test_data/migrate_project.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
is this file test data or code that runs as part of the test?

Reply:
This runs as part of the test to set up the environment/sample app.

Reply:
I was under the impression that this was test data since it's under a directory named `test_data`

Could it be moved to a different directory?

Reply:
This generates/gets the test fixtures, which is consistent with what the other files in this directory do.

Reply:
ok. This seems like a different pattern. Usually test data doesn't contain code that is executed to generate test data or fixtures.